Upstatement/skela-wp-theme
# Add M1 Compatibility & Pull in 2022 ESK updates

## Context 

Was trying to run the ups.com repo (which is based on Skela) and hit some snags going through the setup steps with my M1 mac!

## Changes

Updates from:
https://github.com/Upstatement/editorial-starter-wp-theme/pull/179/files
https://github.com/Upstatement/editorial-starter-wp-theme/pull/190
https://github.com/Upstatement/editorial-starter-wp-theme/pull/191
and others

## How To Test

With an M1 mac go through installation steps https://github.com/Upstatement/skela-wp-theme#-installation

- `./bin/uninstall` (if you've installed skela before)
- `nvm install`
- Duplicate the contents of `.env.sample` into a new `.env` file if you don't already have it
- `./bin/install`
- Confirm installation is successful then `./bin/start` and confirm the following works
  - http://skela.ups.dock and http://skela.ups.dock/wp-admin (`admin`/`password`)
  - http://localhost:3001 and http://localhost:3001/wp-admin (`admin`/`password`)
  - And skela shows up in your ups dock dashboard http://ups.dock/
    <img width="345" alt="image" src="https://user-images.githubusercontent.com/6599979/211057244-8aedcc93-edbe-4ab3-9e83-c00bafb6da6d.png">
